eng/tools/BinaryToolKit/BinaryToolKit.csproj

```diff
@@ -3,10 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
-    <CustomPackageVersionsProps></CustomPackageVersionsProps>
-    <OutputType>Exe</OutputType>
-    <!-- TODO: Re-enable NuGet Audit when BinaryToolKit receives expected global properties: https://github.com/dotnet/dotnet/issues/815 -->
-    <NuGetAudit>false</NuGetAudit>
+    <CustomPackageVersionsProps Condition="'$(BinariesPackagesDir)' != ''">$(BinariesPackagesDir)/PackageVersions.props</CustomPackageVersionsProps>
```

Comment:
Is this still necessary even though we have https://github.com/dotnet/dotnet/blob/f260dfb94fd4c5012022e93d62c84696b74cc9ac/Directory.Build.props#L49C22-L49C67 now? Maybe because it's running in prep-source-build.sh, before we extract PSB?